tests/RobotFramework/tests/cumulocity/telemetry/child_device_telemetry.robot

```diff
@@ -237,6 +237,7 @@ Nested child devices support sending event
  
 # Nested child device services 
 Nested child device service support sending simple measurements
+    Skip    msg=Flaky test (#2416)
```

Comment:
Ignoring this test for now because it's flaky.

Reply:
Please don't skip the test in a unrelated PR.

Reply:
I'll monitor it and retrigger the systems tests if needed (even though it is annoying)

Reply:
fixed in c0477e5